### ISSUE TITLE
Azure Container "No Frameworks Found"

### ISSUE THREAD

**Issue by DannyGoodayle** [dotnet/aspire#1847]
Hey there,

After running `azd up` on small sample project I am running into an issue where the deployed containers don't seem to have the correct framework installed (or any frameworks at all?)

How do I define the framework that the container should use?
My `global.json` currently looks like:

```
{
    "sdk": {
        "version": "8.0.200-preview.23624.5"
    }
}
```

Which seems to do nothing. 

Locally everything is compiling and running, however, I notice that the services aren't being containerized locally so it's fairly difficult to debug. 

I've run `dotnet run --project .\Project.AppHost\Project.AppHost.csproj  --publisher manifest --output-path C:\manifest.json` and can only see `project.v0` for the service definition, not sure how that turns into a container.

**Comment by DannyGoodayle** [comment]
This looks to be related to https://github.com/Azure/azure-dev/issues/3218.

I can confirm that setting the `global.json` in the project root folder to the following:
```
{
    "sdk": {
        "version": "8.0.100"
    }
}
```
Allows the containers to be provisioned correctly. If you find this ticket after searching for the semi-cryptic error message, do the above.

I'll keep this ticket open because it's very difficult to debug. 
- Are there any flags I can use to output the docker containers? 
- It's not obvious that you need to set a specific SDK version to make functional containers, is there any messaging that can be done if an invalid SDK version is set?

**Comment by baronfel** [comment]
Hey folks - this was a point-in-time bug with the SDK Containerization feature that we found and fixed during the latter part of the 8.0.200 development cycle. This shouldn't be an issue on the stable release, or at any point going forward. Please do reopen if you see it happening again.
